ArduCopter/control_rtl.cpp

```diff
@@ -85,7 +85,7 @@ void Copter::rtl_climb_start()
     wp_nav.wp_and_spline_init();
 
     // RTL_SPEED == 0 means use WPNAV_SPEED
-    if (!is_zero(g.rtl_speed_cms)) {
+    if (!is_zero((float)g.rtl_speed_cms)) {
```

Comment:
It's an AP_Int16. This should be changed to

```c++
if (g.rtl_speed_cms.get() != 0)
```

Depending on the case (not here)... calling the cast_to_float() method is another option, too.